src/Util/Image.h

```diff
@@ -128,32 +224,102 @@ struct PointXy {
     }
 };
 
+/**
+ * @struct AxesInfo
+ * @brief Stores information about image axes, including spatial and spectral components.
+ *
+ * This struct holds indices for different axes used in image rendering, spatial mapping,
+ * spectral representation, and stokes parameters.
+ */
 struct AxesInfo {
-    int x, y, spatial_x, spatial_y, spectral, z, stokes;
+    int x;         ///< Index of the x-axis for rendering.
+    int y;         ///< Index of the spatial x-axis.
+    int spatial_x; ///< Index of the spatial x-axis.
+    int spatial_y; ///< Index of the spectral y-axis.
+    int spectral;  ///< Index of the spectral axis.
```

Comment:
This should also be "if applicable" -- we always pass in an explicit value for this, but it may be -1 if there is no spectral axis.

Reply:
Looks like this was added to the wrong line (the spatial Y-axis)